misk-clustering-dynamodb/src/main/kotlin/misk/clustering/dynamo/DynamoCluster.kt

```diff
@@ -6,6 +6,7 @@ import misk.clustering.Cluster.Snapshot
 import misk.clustering.ClusterHashRing
 import misk.clustering.ClusterWatch
 
+@Deprecated("Use DefaultCluster")
```

Comment:
I think since module is new and (probably?) not in use, we should just delete this file if we go that route?

Reply:
Yes I was thinking we could also just delete this